Library/Homebrew/patch.rb

```diff
@@ -64,9 +64,12 @@ def contents
   end
 
   def apply
+    Utils.ensure_git_installed!
+
     data = contents.gsub("HOMEBREW_PREFIX", HOMEBREW_PREFIX)
-    cmd = "/usr/bin/patch"
-    args = %W[-g 0 -f -#{strip}]
+    cmd = "/usr/bin/git"
```

Comment:
Believe we also permit whatever `git` happens to fall in the `$PATH` everywhere else, so would be nice to be consistent here.